app/views/content/how-to-apply-for-teacher-training/teacher-training-application.md

```diff
@@ -57,7 +57,10 @@ You can get help with your application from our [teacher training advisers](/tea
 
 ## What course should I apply for? 
 
-Make sure you check which qualification you’ll get through your training course. Some will award [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts) and some QTS with a [postgraduate certificate in education (PGCE)](/train-to-be-a-teacher/what-is-a-pgce).
+Make sure you check which qualification you’ll get through your training course. You'll train towards one of the following qualifications:
+
+* [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts)
+* [postgraduate certificate in education (PGCE)](/train-to-be-a-teacher/what-is-a-pgce) with QTS
```

Comment:
As per other comment @emmaachesongray - we need to say `QTS with PGCE` not vice versa

Reply:
```suggestion
* QTS with a [postgraduate certificate in education (PGCE)](/train-to-be-a-teacher/what-is-a-pgce)
```